docs/modules/ROOT/pages/clc-config.adoc

```diff
@@ -0,0 +1,75 @@
+= clc config
+
+Configuration operations.
+
+Usage:
+
+[source,bash]
+----
+clc config [command] [options]
+----
+
+== Commands
+
+* <<clc-config-import, clc config import>>
+* <<clc-config-list, clc config list>>
+
+== clc config import
+
+Imports configuration from an arbitrary source.
+
+Usage:
+
+[source,bash]
+----
+clc config import [target] [source]
+----
+
+Parameters:
+
+[cols="1m,1a,2a,1a"]
+|===
+|Parameter|Required|Description|Default
+
+|`target`
+|Required
+|Name of the configuration or its path. If a name is used, the configuration is saved to `$CLC_HOME/configs/NAME`
+|N/A
+
+|`source`
+|Required
+|The following configuration sources are supported:
+
+* Viridian Go Client sample zip file, e.g., `hazelcast-cloud-go-sample-client-pr-3814-default.zip`. Use this if you have downloaded the sample yourself.
+* Viridian Go Client sample cURL command line, such as: `curl https://api.viridian.hazelcast.com/client_samples/download/XXX -o hazelcast-cloud-go-sample-client-pr-3814-default.zip`. Use this for CLC to download the sample. Do not forget to wrap the line with quotes.
```

Comment:
https://api.viridian.hazelcast.com/client_samples/download/XXX

Could we give a full example of a url?

Reply:
There's no fixed URL to download the sample. A different URL is generated for every cluster.